configuration/config2/configfiles/buildset.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
```

Comment:
the 2012 copyright date indicates this file has been around since then, but it appears to be new (at least in this location). Are we moving the original buildset, or is this a placeholder, or are we going to have multiple buildset.xmls?

Reply:
This is still the original. I haven't made any changes

Reply:
We will have at least two buildsets for the time being. I am still looking at the long term need to keep buildset. for now I changed the date.

Reply:
ok, I'm actually not sure if the date should remain at 2012 since it's a pre-existing file or not. We're prob better off keeping the established date. of more concern is the fact that we have duplicate files now. Why can't we just use the original buildset only?

Reply:
based on offline conversation with @kenrowland this file is an exact copy of original buildset.xml and is here as placeholder. I'm not crazy about this approach, but for the sake of progress @kenrowland will open a jira to address this in a separate jira.

Reply:
Jira https://track.hpccsystems.com/browse/HPCC-19032 created to ensure this is removed.